Sources/TuistDependencies/SwiftPackageManager/Utils/ManifestMapping/Project+ManifestMapper.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Project+ManifestMapper.swift
+//  TuistDependencies
+//
+//  Created by Shahzad Majeed on 5/23/22.
+//
+
+import ProjectDescription
+import TuistGraph
+
+extension ProjectDescription.Project {
+    /// Maps a TuistGraph.Project instance into a ProjectDescription.Project instance.
+    /// - Parameters:
+    /// - name: Swift Package / Project name
+    /// - settings:  Project Settings
+    /// - targets:  Project Targets
+    /// - configuration: Configure automatic schemes and resource accessors generation
+    /// for Swift Packages i.e ["package_name":  ProjectConfiguration]
+    static func from(
```

Comment:
This is not a manifest mapper, let's just change the logic in the `PackageInfoMapper`. It should just be using the provided options or the default ones